src/main/java/com/library/feature/user/presentation/UserPresentation.java

```diff
@@ -96,4 +100,11 @@ public static void updateUser(){
         User user = new User(id, name, surname, dni, dateInscription);
         updateUserUseCase.execute(user);
     }
+
+    public static void getUsers() {
+        GetUsersUseCase getUsersUseCase = new GetUsersUseCase(new UserDataRepository(
+                new UserFileLocalDataSource()));
+        List<User> users = getUsersUseCase.execute();
+        System.out.println(users.toString() + "\n");
```

Comment:
Println already adds a newline character.

```suggestion
        System.out.println(users.toString());
```